src/core/control/jobs/PreviewJob.cpp

```diff
@@ -96,6 +97,9 @@ void PreviewJob::drawPage() {
             break;
     }
 
+    xoj::View::Context context{cr2, xoj::View::NORMAL_NON_AUDIO, xoj::View::HIDE_CURRENT_EDITING,
+                               xoj::View::NORMAL_COLOR};
```

Comment:
Since we might want to initialize contexts in a variety of settings, I suggest creating a series of helper methods to create "preset" context settings, e.g.

* `xoj::view::Context::createDefault(cr)`
* `xoj::view::Context::createForExport(cr)`

If necessary, the caller can modify the returned context before using it with the view(s).